manifests/profile/networking/firewall.pp

```diff
@@ -21,8 +24,74 @@
     ensure => absent
   }
 
-  resources { 'firewall':
-    purge => true,
+  if $internal_routing == '' {
+    resources { 'firewall':
+      purge => true,
+    }
+  } else {
+    case $internal_routing {
+      'docker': {
+        $input_ignore = [
+        ]
+
+        $output_ignore = [
+          '-j DOCKER',
+        ]
+
+        $forward_ignore = [
+          '-j DOCKER-USER',
+          '-j DOCKER-ISOLATION',
+          '-i docker0',
+          '-o docker0',
```

Comment:
Docker has the same issue (but different rules) when being run on its own. I don't know how much we'll be moving forward with docker-without-kubernetes, but I figured I may as well express this here.